### ISSUE TITLE
Stopped working after Android UI 3 update 

### ISSUE THREAD

**Issue by Dedos10** [HearthSim/Arcane-Tracker#164]
Stopped working on samsung Galaxy s10 after Android UI 3 update yesterday

**Comment by martinbonnin** [comment]
Hi 👋 . The latests Android versions do not allow to read the Hearthstone logs. 
You can read more at:
https://github.com/HearthSim/Arcane-Tracker/issues/153
and 
https://articles.hsreplay.net/2020/07/22/update-on-arcane-tracker/

**Comment by Dedos10** [comment]
![SmartSelect_20210206-130326_Software update](https://user-images.githubusercontent.com/78643766/107116430-dd6b6080-687b-11eb-84c5-688f0a5f2332.jpg)

**Comment by Dedos10** [comment]
> Hi 👋 . The latests Android versions do not allow to read the Hearthstone logs.
> You can read more at:
> #153
> and
> https://articles.hsreplay.net/2020/07/22/update-on-arcane-tracker/

@martinbonnin isn't there any work around perhaps a permission I need to grant for tracker to work?

**Comment by martinbonnin** [comment]
Nope, we've turned this in all possible ways. Google doesn't want us to read other apps files.
